### PR TITLE
cmd/snap: include locale when linting description being lower case

### PR DESCRIPTION
**Pull request by anonymouse64**
This helps debug which translation is incorrect and needs to be adjusted when
users run into this and paste the output they see directly to i.e. the forum.

### CODE REVIEW

File: i18n/i18n.go [hunk 1]
```diff
@@ -83,10 +83,7 @@ func bindTextDomain(domain, dir string) {
...
 func setLocale(loc string) {
 	if loc == "" {
-		loc = os.Getenv("LC_MESSAGES")
-		if loc == "" {
-			loc = os.Getenv("LANG")
-		}
+		loc = CurrentLocale()
 	}
 	// de_DE.UTF-8, de_DE@euro all need to get simplified
 	loc = strings.Split(loc, "@")[0]
```
Comment (bboozzoo) [inline i18n/i18n.go:89]: WIth the code changes, this is already done in CurrentLocale()

File: i18n/i18n.go [hunk 2]
```diff
@@ -95,6 +92,18 @@ func setLocale(loc string) {
 	locale = translations.Locale(loc)
 }
 
+func CurrentLocale() string {
```
Comment (bboozzoo) [inline i18n/i18n.go:95]:
```
// CurrentLocale returns the current locale without encoding or variants.
```

**Review by bboozzoo — APPROVED** [review]
LGTM with little tweaks

**Review by mvo5 — APPROVED** [review]
Looks fine, thanks for this (but please apply the feedback from Maciej)

File: i18n/i18n.go (hunk 1 above)
Comment (anonymouse64) [reply i18n/i18n.go:94]: I refactored this to be in a shared helper so we don't run it multiple times

File: i18n/i18n.go (hunk 2 above)
Comment (anonymouse64) [reply i18n/i18n.go]: added

**Pull request merged**